src/main/java/mekanism/common/item/ItemAtomicDisassembler.java

```diff
@@ -128,8 +130,9 @@ public boolean onBlockStartBreak(ItemStack itemstack, BlockPos pos, EntityPlayer
                     isOre = true;
                 }
             }
-            if (getMode(itemstack) == 3 && isOre && !player.capabilities.isCreativeMode) {
-                Set<Coord4D> found = new Finder(player, stack, new Coord4D(pos, player.world), raytrace).calc();
+            boolean extended = Mekanism.keyMap.has(player, KeySync.EXTENDEDMINING);
+            if (getMode(itemstack) == 3 && (isOre && !player.capabilities.isCreativeMode || extended)) {
+                Set<Coord4D> found = new Finder(player, stack, new Coord4D(pos, player.world), raytrace, extended ? 10 : -1).calc();
```

Comment:
This radius needs to be added as a config option, and an option to enable the extended vein mining to begin with

Reply:
OK did that